x-pack/plugins/graph/public/application.ts

```diff
@@ -55,10 +54,10 @@ import { SavedObjectsStart } from '../../../../src/plugins/saved_objects/public'
 export interface GraphDependencies {
   pluginInitializerContext: PluginInitializerContext;
   core: CoreStart;
+  coreStart: CoreStart;
```

Comment:
NIT: is this needed if we have `core`?

Reply:
Changing to `core` affects several interfaces and files, so I'd trust the refactoring to the code owners.